src/store/actions/updateMinimumDelegationAmount.ts

```diff
@@ -0,0 +1,73 @@
+import BN from 'bn.js'
+import { useState, useCallback, useEffect } from 'react'
+import { useDispatch } from 'react-redux'
+import { ThunkAction } from 'redux-thunk'
+import { Action } from 'redux'
+import { getAccountWallet } from 'store/account/hooks'
+import { fetchUser } from 'store/cache/user/hooks'
+
+import { Status } from 'types'
+import Audius from 'services/Audius'
+import { AppState } from 'store/types'
+
+const MIN_LOADING_SPINNER_MS = 1000 /* ms */
+
+function updateMinimumDelegationAmount(
+  amount: BN,
+  setStatus: (status: Status) => void,
+  setError: (msg: string) => void
+): ThunkAction<void, AppState, Audius, Action<string>> {
+  return async (dispatch, getState, aud) => {
+    setStatus(Status.Loading)
+    try {
+      const state = getState()
+      const wallet = getAccountWallet(state)
+      if (!wallet) {
+        throw new Error(
+          'No account wallet present, unable to update service provider minimum delegation amount'
+        )
+      }
+
+      // Set the min delegation amount for the user
+      const updateDelegation = aud.Identity.updateMinimumDelegationAmount(
+        amount
+      )
+      const timeoutPromise = new Promise(resolve =>
+        setTimeout(resolve, MIN_LOADING_SPINNER_MS)
+      )
+
+      // Wait a minimum of the loading spinner time
```

Comment:
why?

Reply:
yeah I was worried it would spin for fraction of a sec and go away, but forgot that there was a metamask popup for signing the message so can remove